data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/HistogramAggregateAction.java

```diff
@@ -40,13 +40,6 @@ public class HistogramAggregateAction implements AggregateAction {
     private static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX";
     static final String EVENT_TYPE = "event";
     public static final String HISTOGRAM_METRIC_NAME = "histogram";
-    public static final String SUM_KEY = "sum";
-    public static final String COUNT_KEY = "count";
-    public static final String BUCKETS_KEY = "buckets";
-    public static final String BUCKET_COUNTS_KEY = "bucket_counts";
-    public static final String MIN_KEY = "min";
-    public static final String MAX_KEY = "max";
-    public static final String START_TIME_KEY = "startTime";
     public final String countKey;
```

Comment:
My thought was that these variables could be moved to the HistogramConfig class as well, and can be replaced  with a private variable for 

```
private final HistogramAggregateActionConfig histogramConfig;
```

and then the constructor could just set this
```
this.histogramConfig = histogramAggregateActionConfig
```

And then whenever you need any of the variables the getter for that variable can be used.